packages/ra-ui-materialui/src/preferences/Configurable.tsx

```diff
@@ -92,26 +104,55 @@ export const Configurable = (props: ConfigurableProps) => {
                     isEditorOpen && ConfigurableClasses.editorActive
                 )}
                 sx={sx}
+                ref={wrapperRef}
+                onMouseEnter={isEnabled ? handleShowButton : undefined}
+                onMouseLeave={isEnabled ? handleHideButton : undefined}
             >
-                <Badge
-                    badgeContent={
-                        <SettingsIcon
-                            // @ts-ignore
-                            fontSize="12px"
-                        />
-                    }
-                    componentsProps={{
-                        badge: {
-                            title: translate(openButtonLabel),
-                            onClick: handleOpenEditor,
-                        },
-                    }}
-                    color="warning"
-                    invisible={!isEnabled}
-                >
-                    {children}
-                </Badge>
+                {children}
             </Root>
+            <Popover
```

Comment:
I had to move from Badge to Popover because there is no other way to make the customize button break out of its parent container when that container has overflow:hidden